scripts/deployment/liquidity-mining/update-lm.py

```diff
@@ -106,32 +106,31 @@ def updateLMConfig():
     # SOV/rBTC - 25k SOV
     ALLOCATION_POINT_BTC_SOV = 25000 # (WR)BTC/SOV
 
-    # xUSD/rBTC - 25k SOV
-    ALLOCATION_POINT_BTC_XUSD = 25000 # (WR)BTC/XUSD
+    # xUSD/rBTC - 1 SOV
+    ALLOCATION_POINT_BTC_XUSD = 1 # (WR)BTC/XUSD
 
-    ALLOCATION_POINT_I_XUSD =  1 # iXUSD
+    # DLLR/rBTC - 25k SOV
+    ALLOCATION_POINT_BTC_DLLR =  25000 # (WR)BTC/DLLR
 
     ALLOCATION_POINT_DEFAULT = 1 # 12 tokens with 1 alloc point to account: (WR)BTC/USDT1 | (WR)BTC/USDT2 | (WR)BTC/DOC1 | (WR)BTC/DOC2 | (WR)BTC/BPRO1 | (WR)BTC/BPRO2 | (WR)BTC/MOC | (WR)BTC/FISH | (WR)BTC/RIF | (WR)BTC/MYNT | (WR)BTC/BNB | (WR)BTC/ETH
     ALLOCATION_POINT_CONFIG_TOKEN = MAX_ALLOCATION_POINT - ALLOCATION_POINT_BTC_SOV - ALLOCATION_POINT_BTC_XUSD \
-                                     - ALLOCATION_POINT_I_XUSD - ALLOCATION_POINT_DEFAULT * 12
+                                     - ALLOCATION_POINT_BTC_DLLR  - (ALLOCATION_POINT_DEFAULT * 12)
 
     print("ALLOCATION_POINT_CONFIG_TOKEN: ", ALLOCATION_POINT_CONFIG_TOKEN)
 
     print('LiquidityMiningConfigToken:', lm.getPoolInfo(contracts['LiquidityMiningConfigToken']))
-    print('iXUSD:',lm.getPoolInfo(contracts['iXUSD']))
+    print('WRBTC/DLLR:',lm.getPoolInfo(contracts['(WR)BTC/DLLR']))
 
-    #update this before executing
+    # update this before executing
     data = lm.updateTokens.encode_input(
-        [contracts['iXUSD'], contracts['LiquidityMiningConfigToken']],
-        [ALLOCATION_POINT_I_XUSD, ALLOCATION_POINT_CONFIG_TOKEN],
+        [contracts['(WR)BTC/DLLR'], contracts['LiquidityMiningConfigToken']],
+        [ALLOCATION_POINT_BTC_DLLR, ALLOCATION_POINT_CONFIG_TOKEN],
```

Comment:
missing ALLOCATION_POINT_BTC_XUSD
afaik it is not 1 already, only iXUSD is, but not the AMM pool token

Reply:
this is the poolinfo i get for (W)RBTC/XUSD : ('0x6f96096687952349DD5944E0EB1Be327DcdeB705', 25000, 4975556, 5655368276681260)
allocation points = 25000

Reply:
Ah ya, you are right, was seeing the testnet one.
Updated